README.md

```diff
@@ -5,58 +5,97 @@
   - [List of Contributors](#contributors)
   - [Contact](#contact)
 
 ## Introduction
 
+---
 The TrueBlocks docker version allows you to run our backend in a docker container. This will allow you to create an index of 'every appearance of every address anywhere on the chain.'
 
 With this index, your node software (i.e., [Erigon](https://github.com/ledgerwatch/erigon)) becomes an true Ethereum data server. With such a server, you can build truely distributed applications (trueDapps). TrueDapps are completely trust-less, perfectly private, use immutably data naturally, and participate without effort (by the user) in a shared, distributed application environment.
 
 ## Prerequisite
 
+---
 In order to TrueBlocks Docker to work, it must have access to an RPC endpoint with Parity tracing enabled. Erigon, works this way by default.
 
 There are commercially available offerings, but we find these inadequate because of the way TrueBlocks 'hits' the server. Erigon is easy to install, very fast syncing, and blazingly fast at serving data -- if you have an index and know what to ask for. Another great solution is running both Erigon and TrueBlocks on [dAppNode](https://github.com/dappnode) or [Avado](#).
 
 The docker build environment is also required.
 
-## Building
+## Configuration
+
+---
+We try to make Docker version as easy to configure as possible. This is why it is possible to configure both `chifra` (running inside the container) and the build.
+
+### Configuring `chifra`
+
+---
+Because `chifra` supports multiple configuration items, it is best to store them in a file. There is an example file, `env.example`, in this repository and if you choose to name your file `.env`, it will be ignored by Git.
+
+|Item|Default value|Description|
+|----|-------------|-----------|
+|SCRAPER_ARGS|*empty*|Command line arguments passed to scraper|
+|SCRAPER_FILE|*empty*|Contents of a file with scraper arguments|
+|BOOTSTRAP_BLOOM_FILTERS|`true`|If `true`, the container will run `chifra init` downloading bloom filters|
+|BOOTSTRAP_FULL_INDEX|`true`|If `true`, `chifra init` will download full index|
+|RUN_SCRAPER|`true`|Whether or not to run the scraper|
+|TB_CACHEPATH|`/cache`|Path to TrueBlocks' cache|
+|TB_INDEXPATH|`/index`|Path to TrueBlocks' index|
+|TB_DEFAULTCHAIN|`mainnet`|Chain to use if `--chain` option is not supplied|
+|TB_ETHERSCAN_KEY|*empty*|Your Etherscan API key|
+|TB_CHAINS_MAINNET_CHAINID|`1`|Chain ID (for a chain called `mainnet`)|
+|TB_CHAINS_MAINNET_LOCALEXPLORER|`http://localhost:1234`|URL of the local explorer (TrueBlocks Explorer)|
+|TB_CHAINS_MAINNET_PINGATEWAY|`https://ipfs.unchainedindex.io/ipfs/`|Unchained Index pin gateway|
+|TB_CHAINS_MAINNET_REMOTEEXPLORER|`https://etherscan.io`|Remote explorer URL|
+|TB_CHAINS_MAINNET_RPCPROVIDER|`localhost:8545`|RPC provider URL|
+|TB_CHAINS_MAINNET_SYMBOL|`ETH`|Token symbol for a chain called `mainnet`|
+
+You can add more chains to `chifra` by specifying configuration for them in the format:
```

Comment:
Was there any thought of instructing them to edit a config file or is that (a) too complicated, or (b) not the Docker way of doing things?

Reply:
Editing the file is not the best option when it comes to Docker, because it requires rebuilding the container. With the `.env` file, you just change it and then pass as an argument to `docker run`